### PR TITLE
Linux support

### PR DESCRIPTION
**Pull request by marezina**
System.Drawing library replaced with SkiaSharp

### CODE REVIEW

File: SysKit.XCellKit.SampleApp/Demos/LargeTableStreamingBase.cs [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+﻿using SkiaSharp;
+using System.Collections.Generic;
 
 namespace SysKit.XCellKit.SampleApp.Demos
 {
     abstract class LargeTableStreamingBase : DemoBase
     {
         protected const int RowsToStream = 800000;
-        static Font _font = new Font(new FontFamily("Calibri"), 11);
+        private static SKFont _font = new SKFont(SKTypeface.FromFamilyName("Calibri"), 11);
```
Comment (vinkobedek) [inline SysKit.XCellKit.SampleApp/Demos/LargeTableStreamingBase.cs]: Calibri could be a linux only font

File: SysKit.XCellKit/SysKit.XCellKit.csproj [hunk 2]
```diff
@@ -10,5 +10,8 @@
 	<ItemGroup>
 		<PackageReference Include="DocumentFormat.OpenXml" Version="2.14.0" />
 		<PackageReference Include="Microsoft.Windows.Compatibility" Version="5.0.2" />
```
Comment (vinkobedek) [inline SysKit.XCellKit/SysKit.XCellKit.csproj:12]: this is ok? does it work on linux?

Comment (vinkobedek) [inline SysKit.XCellKit/SysKit.XCellKit.csproj]: Microsoft.Windows.Compatibility is okay? does it work on linux? or just builds and has a runtime issue?

File: SysKit.XCellKit/SpreadsheetStyle.cs [hunk 3]
```diff
@@ -21,25 +20,23 @@ public string GetIdentifier()
...
-                var colorRgb = ForegroundColor.Value.ToArgb();
-                sb.Append(colorRgb);
+                sb.Append(ForegroundColor.Value.GetRgbAsInt());
             }
             if (Font != null)
             {
                 var fontid = Font.ToString();
                 sb.Append(fontid);
-                sb.Append(Font.Style);
+                // sb.Append(Font.Style);
```
Comment (vinkobedek) [inline SysKit.XCellKit/SpreadsheetStyle.cs]: what does this do? will we miss the style property?

File: SysKit.XCellKit/SysKit.XCellKit.csproj (hunk 2 above)
Comment (marezina) [reply SysKit.XCellKit/SysKit.XCellKit.csproj]: I removed this package. Also I have tested it on windows and linux, everything works fine without it.

File: SysKit.XCellKit/SpreadsheetStyle.cs (hunk 3 above)
Comment (marezina) [reply SysKit.XCellKit/SpreadsheetStyle.cs]: This code appends font style to style identifier. I fixed it so it works with SkiaSharp library.

File: SysKit.XCellKit.SampleApp/Demos/LargeTableStreamingBase.cs (hunk 1 above)
Comment (marezina) [reply SysKit.XCellKit.SampleApp/Demos/LargeTableStreamingBase.cs]: Calibri is a windows only font. It's referenced only in tests and demo app, so I would leave it as it is.

**Review by vinkobedek — APPROVED** [review]

**Pull request merged**